server/routes/overview/overviewController.ts

```diff
@@ -2,32 +2,35 @@ import { RequestHandler } from 'express'
 import EducationAndTrainingView from './educationAndTrainingView'
 import OverviewView from './overviewView'
 import SupportNeedsView from './supportNeedsView'
+import { CuriousService } from '../../services'
 
 export default class OverviewController {
+  constructor(private readonly curiousService: CuriousService) {}
+
   getOverviewView: RequestHandler = async (req, res, next): Promise<void> => {
     const { prisonNumber } = req.params
     req.session.createGoalForm = undefined
 
     const { prisonerSummary } = req.session
 
-    const view = new OverviewView(prisonerSummary, 'overview', prisonNumber)
+    const view = new OverviewView(prisonerSummary, prisonNumber)
     res.render('pages/overview/index', { ...view.renderArgs })
   }
 
   getSupportNeedsView: RequestHandler = async (req, res, next): Promise<void> => {
-    const { prisonNumber } = req.params
     const { prisonerSummary } = req.session
     const { supportNeeds } = req.session
 
-    const view = new SupportNeedsView(prisonerSummary, 'support-needs', prisonNumber, supportNeeds)
+    const view = new SupportNeedsView(prisonerSummary, supportNeeds)
     res.render('pages/overview/index', { ...view.renderArgs })
   }
 
-  getEducationAndTrainingNeeds: RequestHandler = async (req, res, next): Promise<void> => {
+  getEducationAndTrainingView: RequestHandler = async (req, res, next): Promise<void> => {
     const { prisonNumber } = req.params
     const { prisonerSummary } = req.session
 
-    const view = new EducationAndTrainingView(prisonerSummary, 'support-needs', prisonNumber)
+    const functionalSkills = await this.curiousService.getPrisonerFunctionalSkills(prisonNumber, req.user.username)
```

Comment:
Here is where we get the functional skills from the curious service, right where we need it in respect of adding it to the view. It is not added to the session or cached in any way.

(I only mention caching here because I suspect we might need to do it one day, but at the moment it would be a premature optimisation; and I'm not convinced adding it to the session would be the right way to do it in this case)